lib/Doctrine/DBAL/Id/TableGenerator.php

```diff
@@ -139,7 +140,7 @@ public function nextValue($sequenceName)
             $this->conn->commit();
         } catch (Throwable $e) {
             $this->conn->rollBack();
-            throw new DBALException('Error occurred while generating ID with TableGenerator, aborted generation: ' . $e->getMessage(), 0, $e);
+            throw new DBALException(sprintf('Error occurred while generating ID with TableGenerator, aborted generation with error "%s".', $e->getMessage()), 0, $e);
```

Comment:
Let's find some consensus on message chaining. If the chained exception also contains another exception, the nested quotes will look ugly:

> Authentication failed with error "Database connection failed with error: "Invalid user name"".

With colons, it will look better:

> Authentication failed: Database connection failed: Invalid user name

When a message chains another message, it probably shouldn't add its own period since the period will be contained in the innermost message.

Reply:
Agree. Colons work better. I will make the changes :+1: